README.txt

```diff
@@ -27,14 +27,14 @@ JDBCSample is a sample jdbc client project. You can compile by executing
 
 and then you execute with the following command. Change as needed for a linux based invocation. 
 
-java -cp .;"<your_path>\kinetica-jdbc-7.0.0.0-jar-with-dependencies.jar" com.gisfederal.gpudbquery.JDBCSample -DriverClass com.kinetica.jdbc.Driver -URL "jdbc:kinetica://gpudb-machine-name-or-ip:9191" -PWD pwd -UID uid
+java -cp .;"<your_path>\kinetica-jdbc-7.1.0.0-jar-with-dependencies.jar" com.gisfederal.gpudbquery.JDBCSample -DriverClass com.kinetica.jdbc.Driver -URL "jdbc:kinetica://gpudb-machine-name-or-ip:9191" -PWD pwd -UID uid
```

Comment:
@jkauffman503, I think it would be useful to put 7.1.x.y with a clariying note that the x & y need to be replaced with actual values.  That would address Andy's concern, too.  Thanks!